Source/Tools/LinuxEmulation/LinuxSyscalls/SyscallsVMATracking.h

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT
+#pragma once
+
+#include <cstdint>
+#include <tuple>
+
+#include <FEXCore/fextl/map.h>
+#include <FEXCore/Utils/SignalScopeGuards.h>
+
+namespace FEXCore::IR {
+struct AOTIRCacheEntry;
+}
+
+namespace FEX::HLE::VMATracking {
+///// VMA (Virtual Memory Area) tracking /////
+
+struct SpecialDev {
```

Comment:
This is pre-existing since the previous code, but is this intended to be a namespace rather than a struct?

Reply:
Probably?

Reply:
switched it to a namespace.